src/code42cli/cmds/search/extraction.py

```diff
@@ -94,16 +94,10 @@ def handle_response(response):
         total_events = len(events)
         handlers.TOTAL_EVENTS += total_events
 
-        def _format_output():
-            return formatter.get_formatted_output(events)
-
         if total_events > 10 or force_pager:
-            click.echo_via_pager(_format_output())
+            click.echo_via_pager(events)
```

Comment:
This should still call format on the events, otherwise it's echoing the dicts instead of json.